### DISCUSSION TITLE
Saturated colours feels much brighter than non saturated colours at 100% colour normalisation

### DISCUSSION THREAD

**Discussion by huupoke** [ppy/osu#21275] [Q&A] [answered]
The new colour normalisation setting is great, but even at 100%, it still makes high saturated colours much brighter than low saturated colours. The problem can be seen more easily when playing (I don't upload video because I think the compression will change the colours).

* Beatmap: https://osu.ppy.sh/beatmapsets/219380#osu/541392
* High saturated colour:
![osu_2022-11-17_19-00-47](https://user-images.githubusercontent.com/16283764/202442053-8a106f1f-3df5-4dfa-a363-908eed3a003f.png)
* Low saturated colour:
![osu_2022-11-17_19-00-45](https://user-images.githubusercontent.com/16283764/202442069-88826eb6-39dd-46f1-8f73-10f959ba2f22.png)

**Comment by smoogipoo** [comment, marked as answer]
These look about the same to me overlaying them in image editing software. As always, colour is subjective and so is this algorithm, but you can read more about how it works here: https://alienryderflex.com/hsp.html osu! normalises to 60% brightness when that slider is at 100%.
